frontend/src/main/web/lib/components/glossary/ActionCell.jsx

```diff
@@ -51,8 +55,34 @@ var ActionCell = React.createClass({
     Actions.resetEntry(this.props.id);
   },
 
-  _onUpdateComment: function (value) {
-    Actions.updateComment(this.props.id, value);
+  _onUpdateComment: function () {
+    this.setState({savingComment: true});
+    Actions.updateComment(this.props.id, this.state.comment);
+  },
+
+  _hasCommentChanged: function() {
+    var initialValue = _.isUndefined(this.state.entry.transTerm.comment) ? '' : this.state.entry.transTerm.comment;
+    var newValue = _.isUndefined(this.state.comment) ? '' : this.state.comment;
```

Comment:
Little module for this pattern: [defined](https://www.npmjs.com/package/defined). It uses the first value that is not `undefined`. You can do:

```javascript
let newValue = defined(this.state.comment, '')
```